### PR TITLE
Add tunnelblick support

### PR DESCRIPTION
**Pull request by pomdtr**
## Description

This commands allow to quickly interact with tunnelblick vpn

## Type of change

- [x] New script command

## Screenshot

image

## Dependencies / Requirements

Tunnelblick App

## Checklist

- [x] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

**Comment by imthath-m** [comment]
Hey @pomdtr , Thanks for the contribution. 

It looks good but we generally use title-case for commands and arguments. Can you change the `configuration` argument to `Configuration`?

File: commands/apps/tunnelblick/tunnelblick-connect.applescript [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Tunnelblick: Connect
+# @raycast.mode silent
+# @raycast.packageName Tunnelblick
+#
+# Optional parameters:
+# @raycast.icon images/tunnelblick.png
+# @raycast.needsConfirmation false
+# @raycast.argument1 { "type": "text", "placeholder": "configuration" }
```
Comment (imthath-m) [inline commands/apps/tunnelblick/tunnelblick-connect.applescript:12]:
```suggestion
# @raycast.argument1 { "type": "text", "placeholder": "Configuration" }
```

File: commands/apps/tunnelblick/tunnelblick-disconnect.applescript [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Tunnelblick: Disconnect
+# @raycast.mode silent
+# @raycast.packageName Tunnelblick
+#
+# Optional parameters:
+# @raycast.icon images/tunnelblick.png
+# @raycast.needsConfirmation false
+# @raycast.argument1 { "type": "text", "placeholder": "configuration" }
```
Comment (imthath-m) [inline commands/apps/tunnelblick/tunnelblick-disconnect.applescript:12]:
```suggestion
# @raycast.argument1 { "type": "text", "placeholder": "Configuration" }
```

Comment (imthath-m) [inline commands/apps/tunnelblick/tunnelblick-disconnect.applescript]:
```suggestion
# @raycast.argument1 { "type": "text", "placeholder": "Configuration" }
```

**Comment by imthath-m** [comment]
@pomdtr, I've made the changes and merged it. You can ensure to use title case for arguments next time around.

**Comment by pomdtr** [comment]
Thanks !

**Pull request merged**